Makefile

```
@@ -115,3 +115,7 @@ test-unused-modules:
 .PHONY: test-soft_failure-no-reference
 test-soft_failure-no-reference:
 	@! git --no-pager grep -E -e 'soft_failure\>.*\;' --and --not -e '([$$0-9a-z]+#[$$0-9]+|fate.suse.com/[0-9]|\$$[a-z]+)' lib/ tests/
+
+.PHONY: test-invalid-syntax
+test-invalid-syntax:
+	tools/tools/check_invalid_syntax
```

As an alternative to no add yet another script, this can be rewritten as: 
```suggestion
	tools/check_invalid_syntax
```